### DISCUSSION TITLE
isn't `syncPeriod` should also be changed for webhookServer?

### DISCUSSION THREAD

**Discussion by erplsf** [actions/actions-runner-controller#1403] [Questions] [answered]
Hi. I saw in release notes change to default `syncPeriod` in helm charts, but I also noticed the similar variable for `webhookServer`. Do they have different meaning?

**Comment by toast-gear** [comment, marked as answer]
it's dead code, we have a ticket to remove it https://github.com/actions-runner-controller/actions-runner-controller/issues/623, just haven't got around to it yet. The webhook server went through a number of implementations
